templates/footer.hbs

```diff
@@ -1,4 +1,5 @@
 {{#if varsb.javascript_highlightjs}}<script type="text/javascript" charset="utf-8">hljs.initHighlighting();</script>{{/if}}
 <script type="text/javascript" src="/menu.js?{{cratesfyi_version_safe}}"></script>
+<script type="text/javascript" src="/index.js"></script>
```

Comment:
This will prevent the page from being cached indefinitely.
```suggestion
<script type="text/javascript" src="/index.js?{{cratesfyi_version_safe}}"></script>
```